Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,8 @@
 
 /* Begin PBXBuildFile section */
 		08222B07C311489F90D08D41 /* Pods_Demo.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6D23244B5E9EE59BAB3F3003 /* Pods_Demo.framework */; };
+		08A83F592EDE268E0076BD28 /* VenmoPaymentButtonViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 08A83F582EDE268E0076BD28 /* VenmoPaymentButtonViewController.swift */; };
```

Comment:
Depending on our other decision on naming lets rename this to just PaymentButtonViewController.